src/mime.cpp

```diff
@@ -39,11 +39,15 @@
 #include <QHash>
 #include <QDBusInterface>
 #include <QDBusPendingCall>
-
+#include <QRegularExpression>
 #include <MDesktopEntry>
 
 #include <sys/stat.h>
 
+#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
```

Comment:
Qt 5.14 but thinking if all could be just std::endl which I think was getting effectively used here so far.